app/views/locations/_render_location_detail.html.haml

```diff
@@ -29,7 +29,7 @@
 				Location Permalink
 	%div.m_10[l, :metadata]
 		= render :partial => 'locations/render_update_metadata', :locals => {:l => l}
-	- if ENV['AWS_BUCKET_NAME'] && !is_bot?
+	- if ENV['AWS_BUCKET_NAME'].present? && !is_bot?
```

Comment:
I was getting a lot of errors in my local env related to this, so I just went and made it optional